conn-sdk-cli/main.go

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/conduitio/conduit-connector-sdk/conn-sdk-cli/readmegen"
+	"github.com/conduitio/conduit-connector-sdk/conn-sdk-cli/specgen"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	cmdRoot := &cobra.Command{
```

Comment:
WDYT about trying ecdysis here too?

Reply:
We could of course do that, I'm not sure if it's worth it now though. I went with vanilla cobra here as I needed something simple for 2 commands only, while ecdysis shines in organizing commands in bigger CLIs IMO.

Reply:
Makes sense :+1: .